lib/bdd/rspec/formatter.rb

```diff
@@ -0,0 +1,83 @@
+RSpec::Support.require_rspec_core "formatters/documentation_formatter"
```

Comment:
I just moved this file and removed the monkey patch approach. The rest are all the same.